assets/themes/dog/dog.json

```diff
@@ -0,0 +1,427 @@
+{
+  "id": "dog",
+  "maintainer": "Niels Elgaard Larsen",
+  "version": "1",
+  "title": {
+    "en": "dog POIs",
+    "da": "hundesteder"
+  },
+  "description": {
+    "en": "Hundsteder",
+    "da": "Hundsteder"
+  },
+  "icon": "./assets/themes/dog/dog-park.svg",
+  "startZoom": 10,
+  "startLat": 55,
+  "startLon": 12,
+  "layers": [
+    {
+      "id": "dogpark",
+      "source": {
+        "osmTags": {
+          "#": "For a description on which tags are possible, see https://github.com/pietervdvn/MapComplete/blob/develop/Docs/Tags_format.md",
+          "or": [
+            "leisure=dog_park",
+            {
+              "and": [
+                "leisure=park",
+                "dog=unleashed"
+              ]
+            }
+          ]
+        }
+      },
+      "minzoom": 10,
+      "presets": [
+        {
+          "title": {
+            "en": "a dog park",
+            "da": "en hundeskov"
+          },
+          "tags": [
+            "leisure=dog_park"
+          ],
+          "description": {
+            "en": "A place for dogs, where they can run unleashed",
+            "da": "En hundeskov eller hundeg&aring;rd beregnet til hunde, hvor de kan l&oslash;be uden snor"
+          }
+        }
+      ],
+      "name": {
+        "en": "dog parks",
+        "da": "hundeskove"
+      },
+      "title": {
+        "render": {
+          "en": "dog park",
+          "da": "hundeskov"
+        },
+        "mappings": [
+          {
+            "if": "name~*",
+            "then": {
+              "*": "{name}"
+            }
+          }
+        ]
+      },
+      "mapRendering": [
+        {
+          "location": [
+            "point",
+            "centroid"
+          ],
+          "icon": "./assets/themes/dog/dog-park.svg",
+          "iconSize": "40,40,center",
+          "label": {
+            "mappings": [
+              {
+                "if": "name~*",
+                "then": "<div style='background: white; padding: 0.25em; border-radius:0.5em'>{name}</div>"
+              }
+            ]
+          }
+        },
+        {
+          "#": "Rendering of a line",
+          "color": "#ff0",
+          "width": 5
+        }
+      ],
+      "tagRenderings": [
+        {
+          "id": "dogpark-fenced",
+          "mappings": [
+            {
+              "if": "barrier=fence",
+              "then": {
+                "en": "fenced: Yes",
```

Comment:
If the property is already known, this will be shown 'as is' to the user. It is better to use a full sentence, e.g. "This dogpark is fenced all around" instead of "fenced: yes". Same for "fenced: no"